prow/gerrit/projects/projects.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package projects defines the flag for initializing a gerrit client
```

Comment:
actually does this even belong in it's own package rather than just part of gerrit? /shrug
reading through the code it seems this is just a complex flag type? /shrug

Reply:
/shrug

ok, moving

Reply:
done I think